packages/page-council/src/Motions/useVotingStatus.ts

```diff
@@ -18,20 +20,24 @@ interface State {
 function getStatus (api: ApiPromise, bestNumber: BlockNumber, votes: Votes, numMembers: number): State {
   if (!votes.end) {
     return {
+      hasFailed: false,
+      hasPassed: false,
       isCloseable: false,
       isVoteable: true,
       remainingBlocks: null
     };
   }
 
   const isEnd = bestNumber.gte(votes.end);
-  const isPassing = votes.threshold.lten(votes.ayes.length);
-  const isFailing = votes.threshold.gtn(Math.abs(numMembers - votes.nays.length));
+  const hasPassed = votes.threshold.lten(votes.ayes.length);
+  const hasFailed = votes.threshold.gtn(Math.abs(numMembers - votes.nays.length));
```

Comment:
shouldn't `hasPassed` always be `!hasFailed` (and vice versa?)

Reply:
No. It will always map to what is in substrate -

https://github.com/paritytech/substrate/blob/master/frame/collective/src/lib.rs#L658-L659

Reply:
You're right, nevermind :facepalm: